### ISSUE TITLE
Customise licence notice

### ISSUE THREAD

**Issue by ferrine** [lucianopaz/head_of_apache#1]
Is it possible use a different licence text? Maybe a cli argument that will use non default text template?

**Comment by lucianopaz** [comment]
Since this project only tried to apply the Apache v2 license header to files, I didn't make it possible to customize the text beyond setting the author. What did you have in mind?

**Comment by ferrine** [comment]
So not only the apache option is available but arbitrary one, MIT, Custom, etc. Should be pretty simple to get the template from a local file and apply it as is.

**Comment by ferrine** [comment]
That case, maybe the appropriate name for the hook becomes 'licence_head'

**Comment by lucianopaz** [comment]
Yes, it could be done with ease. It could rely on jinja2 or even simple string interpolation. I’ll give this some thought after the new year
